### ISSUE TITLE
Wing Commander 3 freezes during video transition scenes

### ISSUE THREAD

**Issue by skinlab** [libretro/opera-libretro#175]
Random hard locks when moving from room to room around the ship.

**Comment by tacogerbil** [comment]
I'm seeing this too with Wing commander III Heart of the Tiger on Opera 1.16.0 (1.0.0 100ae1e) using a steam deck.  
I've tried CHD and bin/cue, no difference on the internal SSD.  Even overclocking it doesn't make a difference.  Every FMV seems to have a random chance of locking up the game.  
I can still access the Retroarch menu so it's not a hard lock on the system.

**Comment by trapexit** [comment]
Yes, I've confirmed the issue. It's unclear what's going on. The 3DO OS appears to still be running normally suggesting the player itself is freezing up. This makes it a lot harder to investigate.
